package.json

```diff
@@ -136,11 +136,11 @@
     "stylelint": "^8.4.0",
     "stylelint-config-standard": "^18.0.0",
     "stylelint-processor-styled-components": "^1.2.2",
-    "ts-jest": "^22.0.2",
+    "ts-jest": "^22.4.6",
     "tslint": "^5.8.0",
     "tslint-config-prettier": "^1.6.0",
-    "typescript": "^2.5.3",
-    "typescript-styled-plugin": "^0.1.2"
+    "typescript": "2.6.2",
```

Comment:
this is the specific version we were on, updating any further starts to break the types in RN and SC